cli/asc.js

```diff
@@ -422,6 +422,7 @@ exports.main = function main(argv, options, callback) {
   assemblyscript.setPedantic(compilerOptions, opts.pedantic);
   assemblyscript.setLowMemoryLimit(compilerOptions, opts.lowMemoryLimit >>> 0);
   assemblyscript.setExportRuntime(compilerOptions, opts.exportRuntime);
+  assemblyscript.setBundleVersion(compilerOptions, exports.version);
```

Comment:
It would be good if `asc.js` already decomposed the version into its components, so we do not need to pass a string here but just numbers. As written, this should currently not pass the correct value to the compiler compiled to Wasm.

Reply:
Done! Thanks for your comments!